### PR TITLE
bodyValidation file

### PR DESCRIPTION
**Pull request by Mirak005**
separate field validation with express-validator ( trying to contribute  :p :p :p)

### CODE REVIEW

File: middleware/bodyValidation.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+const { body, validationResult } = require('express-validator');
+
+//Register Form
+const registerRules = () => [
```
Comment (bushblade) [inline middleware/bodyValidation.js:4]: This doesn't need to be a function, you can have arrays of middleware in a express route handler, so for example you could simply do...
```js
const registerRules = [
  body('name', 'Name is required ').notEmpty(),
  body('email', 'Please include a valid email').isEmail(),
  body(
    'password',
    'Please enter a password with 6 or more characters'
  ).isLength({
    min: 6,
    max: 20
  })
];
```
No need for a function.

**Comment by bushblade** [comment]
Thanks for the PR but I don't think this solves anything and would probably make it a little more confusing for students migrating their code from the course code.

File: middleware/bodyValidation.js (hunk 1 above)
Comment (Mirak005) [reply middleware/bodyValidation.js:4]: Yep you're right , unnecessary functions  , thx for you reply , i will fix that  :D

Comment (bushblade) [reply middleware/bodyValidation.js:4]: No worries. I do like the idea but yeah I don't think we _need_ this.

**Pull request closed without merging**